lib/js/components/Badges/Badge/Badge.vue

```diff
@@ -1,6 +1,13 @@
 <template>
-	<div class="badge" :class="{ '-neutral': color === BADGE_COLORS.NEUTRAL }">
-		<div class="badge__slot">
+	<div
+		class="badge"
+		:class="{
+			'-neutral': color === BADGE_COLORS.NEUTRAL,
+			'-primary': color === BADGE_COLORS.PRIMARY,
+		}"
+	>
+		<ds-icon v-if="icon" class="badge__icon" :icon="icon" :size="ICON_SIZES.XX_SMALL" />
+		<div v-if="$slots.default" class="badge__slot">
```

Comment:
UX mentioned that these icons should not be "either or", these can be displayed simultaneously - https://miro.com/app/board/o9J_knPnexA=/?moveToWidget=3458764569145169599&cot=14

Reply:
Since these can appear simultaneously, I've added a story for a case where there is no slot used - https://github.com/bethinkpl/design-system/pull/327/commits/e6b3a1315033fa58722a42093e06e6e896e0ba27